File: python/1019.py
n = int(input())


def conversão(numero_segundos):
    horas = numero_segundos // 3600
    minutos = (numero_segundos % 3600) // 60
    segundos = (numero_segundos % 3600) % 60

    return "%d:%01d:%01d" % (horas, minutos, segundos)


print(conversão(n))

File: c/1019.c
#include <stdio.h>

int main() {

    int segundos_total, horas, resto, minutos, segundos;
    
    scanf("%d", &segundos_total);
 
    horas = segundos_total / 3600;
    resto = segundos_total % 3600;
    minutos = resto / 60;
    resto = resto % 60;
    segundos = resto;

    printf("%d:%d:%d\n", horas, minutos, segundos);
   
    return 0;
}